storagenode/pieces/store.go

```diff
@@ -58,6 +58,8 @@ type DB interface {
 	Add(context.Context, *Info) error
 	// Get returns Info about a piece.
 	Get(ctx context.Context, satelliteID storj.NodeID, pieceID storj.PieceID) (*Info, error)
+	// GetAll returns Info about pieces held for a satellite
```

Comment:
This comment should be fixed to match the name and functionality of the `GetPiecesID`.